<ui version="4.0" >
 <class>ClipartWindowClass</class>
 <widget class="QWidget" name="ClipartWindowClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>353</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="styledFrame" >
     <property name="acceptDrops" >
      <bool>true</bool>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth" >
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4" >
      <property name="spacing" >
       <number>0</number>
      </property>
      <property name="leftMargin" >
       <number>1</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>1</number>
      </property>
      <property name="bottomMargin" >
       <number>1</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <property name="topMargin" >
         <number>5</number>
        </property>
        <property name="rightMargin" >
         <number>4</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_4" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>10</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="closeToolButton" >
          <property name="focusPolicy" >
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip" >
           <string>CloseButtonHint</string>
          </property>
          <property name="accessibleName" >
           <string>transparentButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex" >
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab" >
         <attribute name="title" >
          <string>ClipartTabText</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_20" >
            <property name="bottomMargin" >
             <number>0</number>
            </property>
            <item>
             <widget class="MoodBox::ClipartCategoriesList" native="1" name="clipartCategorySelector" />
            </item>
           </layout>
          </item>
          <item>
           <widget class="MoodBox::ClipartEntriesList" name="clipartEntriesList" >
            <property name="verticalScrollBarPolicy" >
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="horizontalScrollBarPolicy" >
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="autoScroll" >
             <bool>false</bool>
            </property>
            <property name="editTriggers" >
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="dragDropMode" >
             <enum>QAbstractItemView::DragDrop</enum>
            </property>
            <property name="iconSize" >
             <size>
              <width>96</width>
              <height>96</height>
             </size>
            </property>
            <property name="verticalScrollMode" >
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="flow" >
             <enum>QListView::LeftToRight</enum>
            </property>
            <property name="isWrapping" stdset="0" >
             <bool>true</bool>
            </property>
            <property name="resizeMode" >
             <enum>QListView::Adjust</enum>
            </property>
            <property name="viewMode" >
             <enum>QListView::IconMode</enum>
            </property>
            <property name="uniformItemSizes" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2" >
         <attribute name="title" >
          <string>PhotoTabText</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" >
            <property name="topMargin" >
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="folderLabel" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="alignment" >
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="readOnly" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="browseButton" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="accessibleName" >
               <string>rectButton</string>
              </property>
              <property name="text" >
               <string>BrowseButtonText</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="MoodBox::ClipartEntriesList" name="photoList" >
            <property name="verticalScrollBarPolicy" >
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="horizontalScrollBarPolicy" >
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="autoScroll" >
             <bool>false</bool>
            </property>
            <property name="editTriggers" >
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="dragDropMode" >
             <enum>QAbstractItemView::DragOnly</enum>
            </property>
            <property name="iconSize" >
             <size>
              <width>96</width>
              <height>96</height>
             </size>
            </property>
            <property name="verticalScrollMode" >
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="flow" >
             <enum>QListView::LeftToRight</enum>
            </property>
            <property name="isWrapping" stdset="0" >
             <bool>true</bool>
            </property>
            <property name="resizeMode" >
             <enum>QListView::Adjust</enum>
            </property>
            <property name="uniformItemSizes" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3" >
         <attribute name="title" >
          <string>BackgroundTabText</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_40" >
          <item>
           <widget class="MoodBox::ClipartEntriesList" name="tileList" >
            <property name="verticalScrollBarPolicy" >
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="horizontalScrollBarPolicy" >
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="autoScroll" >
             <bool>false</bool>
            </property>
            <property name="editTriggers" >
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="dragDropMode" >
             <enum>QAbstractItemView::DragOnly</enum>
            </property>
            <property name="iconSize" >
             <size>
              <width>96</width>
              <height>96</height>
             </size>
            </property>
            <property name="verticalScrollMode" >
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="flow" >
             <enum>QListView::LeftToRight</enum>
            </property>
            <property name="isWrapping" stdset="0" >
             <bool>true</bool>
            </property>
            <property name="resizeMode" >
             <enum>QListView::Adjust</enum>
            </property>
            <property name="uniformItemSizes" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" >
        <property name="spacing" >
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" native="1" name="widget_2" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" native="1" name="widget" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Minimum" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>MoodBox::ClipartEntriesList</class>
   <extends>QListView</extends>
   <header>clipartentrieslist.h</header>
  </customwidget>
  <customwidget>
   <class>MoodBox::ClipartCategoriesList</class>
   <extends>QWidget</extends>
   <header>clipartcategorieslist.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
